about.html

```diff
@@ -0,0 +1,97 @@
+                    <li><a href="">Join</a></li>
+                    <li><a href="">Sponsor</a></li>
+                    <li><a href="">News</a></li>
+                    <li><a href="">Campiagn</a></li>
+                </ul>   
+            </div>
+        </div>
+    </nav>
+    <header>
+        <div class="about-info container about-headline primary-font">
+            <button class="nav-btn"><i class="fa-solid fa-bars fa-2x"></i></button>
+            <p class="sub-title">"Kabul Mesuem Exhibition"</p>
+            <h1 class="title">FOURTH ROUND OF KABUL MESUEN EXHABITION</h1>
+            <hr class="title-underline">
+            <div class="about-short-info">
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer congue mollis malesuada. Mauris enim massa, interdum dictum vestibulum at, pretium vitae libero. Morbi libero tortor, luctus tincidunt purus et, elementum tempus nibh. Etiam elit nisl, sagittis quis consequat sit amet, congue vitae turpis. In pellentesque aliquam felis eu varius. Pellentesque interdum nisl ut justo tempor tincidunt. Ut sodales vel quam a semper.</p>
+                <p>Duis accumsan venenatis eros. Nullam interdum neque lectus, eget pretium ligula vulputate nec. Proin et ipsum pharetra, lobortis est ut, suscipit nisi. Mauris blandit laoreet nibh. Mauris semper, orci ac iaculis egestas, magna dolor suscipit justo, varius vehicula purus quam quis enim. In consectetur pulvinar dui, ut vestibulum orci rhoncus et. Aliquam nec lacinia lacus.</p>
+            </div>
+            <p>Please contact us per Email for any further questions about Exhibition</p>
+            <a href="">kabulmesuem@afg.org</a>
+        </div>
+    </header>
+    <main>  
+        <section class="about-content primary-font">
+            <div class="container">
+                <h2>Kabul Mesuem Exhibition Event 2023</h2>
+                <hr class="title-underline">
+                <p class="exhibition-info">The logo of kabul Mesuem Exhibition 2023 was decided through the logo competition from 8.June to 7.July.</p>
+                <div class="exhibition-logo">
+                    <img class="afg-logo" src="image/afg-logo.png" alt="">
+                </div>
+            </div>
+            <hr class="container-sep">
+            <div class="past-event container">
+                <h2>See the past Exhibitions</h2>
+                <hr class="title-underline">
+                <p>Take a look the last two Exhabition which took place in Darulaman Kabul.</p>
+                <div class="events">
+                    <div class="event">
+                        <h3>2020</h3>
+                        <p>Kabul Mesuem Exhibition 2020 in Darulman</p>
+                    </div>
+                    <div class="event">
+                        <h3>2022</h3>
+                        <p>Kabul Mesuem Exhibition 2020 in Darulman</p>
+                    </div>
+                </div>
```

Comment:
- [ ] As requested by the previous reviewer the images here should have a red transparent layer on them so as to match the given design. Kindly fix this

Your design
![image](https://user-images.githubusercontent.com/81932478/233795133-c225a553-d861-4194-be1e-9e98ba6261e5.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/233795153-09853dfe-34f2-4186-b05b-eff0243167ef.png)